cli/tests/unit/write_file_test.ts

```diff
@@ -96,6 +96,20 @@ Deno.test(
   },
 );
 
+Deno.test(
+  { permissions: { read: true, write: true } },
+  function writeFileSyncCreateNew() {
+    const enc = new TextEncoder();
+    const data = enc.encode("Hello");
+    const filename = Deno.makeTempDirSync() + "/test.txt";
```

Comment:
We should clean up this temp dir after running the test.

Reply:
We dont do any cleaning up whatsoever with any of the other tests in this file; i would say to add some sort of cleanup it should be a follow-up

Reply:
Sure, a follow up sounds good. We should be cleaning up when running tests.